src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -97,7 +97,12 @@ extension AppDelegate {
 				secondaryActionTitle: AppStrings.Common.errorAlertActionMoreInfo,
 				secondaryActionCompletion: openFAQ
 			)
-		default:
+		} else if let exposureDetectionError = noSummaryError as? ExposureDetectionError {
+			switch exposureDetectionError {
+			case .isAlreadyRunning:
+				return nil
```

Comment:
Doesn't the error also trigger a different cell on the home view? Should we throw this error at all?

Reply:
Very good point! 
Currently it cannot happen, because RiskProvider "guards" this case for the key download and the risk detection.
But I think it is still a valid point from the perspective of code readability.
I have added error handling to HomeInteractor which explicitly ignores the allready running errors.